### PR TITLE
add docs for container protection mode

### PR DESCRIPTION
**Pull request by Crazykev**
@Gnep updated help message and default value.

Signed-off-by: Crazykev <crazykev@zju.edu.cn>

### CODE REVIEW

**Comment by kevinsimper** [comment]
Nice, I have many times rm all my containers because it was easier than figuring out which to keep :D

File: Reference/CLI/create.md [hunk 1]
```diff
@@ -21,6 +21,7 @@
               --noauto-volume                 Do not create volumes specified in image
               -P, --publish-all               Publish all exposed ports
               -p, --publish=[]                Publish a container's port(s)
+              --protection=false              Termination protection for container
```
Comment (Gnep) [inline Reference/CLI/create.md]: I'm not sure about `false`. There is `on/off`, `yes/no`. Better to use a common one. 

@kevinsimper What do you think?

Comment (kevinsimper) [inline Reference/CLI/create.md]: I would prefer true/false, like `sig-proxy` uses :)

Comment (Gnep) [inline Reference/CLI/create.md]: I see. Fair enough

**Pull request merged**